Project/Assets/Prairie/Framework/Script/Interaction/Slideshow.cs

```diff
@@ -1,6 +1,7 @@
 using UnityEngine;
 using System.Collections;
 
+[AddComponentMenu("Prairie/Interactions/Slideshow")]

```

Comment:
Elsewhere, we treat `Slideshow` as an Annotation (its bound to the annotation button in player input) so this should be folded into the "_Prairie/Annotation/_" category.

Just to be clear about a potential confusion, even though I am requesting this change - I still agree that `Slideshow` should derive from `PromptInteraction`.